scripts/tables.R

```diff
@@ -1432,9 +1432,39 @@ tab_map <- tab_map_prep %>%
   dplyr::distinct(site_id, .keep_all = T) #just for now
 
 
+# moti climate change table --------------------------------------------------------------
 
+# read in amalgamated pscis form
+form_id <- paste0(getwd(), '/data/inputs_extracted/form_pscis_v225.gpkg')
```

Comment:
you actually don't need to paste the working directory to the front of the path b/c we are working in projects.  The main directory is ground zero so 

`form_id <- 'data/inputs_extracted/form_pscis_v225.gpkg'`

however, this file is not actually here yet.  It was built in the skeena repo.  we can do it here too though https://github.com/NewGraphEnvironment/fish_passage_skeena_2022_reporting/issues/37